cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,132 @@
 /// <reference types='cypress' />
+const { generateWorker } = require('../support/generateWorker');
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  const {
+    firstName,
+    lastName,
+    email,
+    age,
+    salary,
+    department
+  } = generateWorker();
+
+  it('should contains pagination', () => {
```

Comment:
this test doesn't check the pagination. It just checks text in the pagination component. You need to click buttons and check that pages change